docker-compose.yml

```diff
@@ -3,19 +3,12 @@ services:
   editor:
     build:
       context: .
+      args:
```

Comment:
Here is the mechanism by which we inject environment variables. Presumably there is a Terraform/ECS way of doing the same, @rsmith11?